cli/cli_test.go

```diff
@@ -0,0 +1,29 @@
+package cli
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+	"polydawn.net/repeatr/testutil"
+)
+
+var (
+	// os flag parsing mandates the executable name
+	baseArgs = []string{"repeatr"}
+)
+
+// These can be swapped wholesale; demonstration only.
+
+func Test(t *testing.T) {
+
+	Convey("It should not crash without args", t, func() {
+		App.Run(baseArgs)
```

Comment:
We should rig this up to have the os in/out/err be parameters passed all the way through so we can shunt this output to dev null (or the capture mechanism of goconvey so it shows up nicely in reports).